### PR TITLE
feat: Terms of use web view

### PR DESCRIPTION
**Pull request by TikuGaelle**
## Description

- Terms of use web view page with internationalization
- Fixes issue #27 

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

## Pre-launch Checklist

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] I read and ran all relevant commands as specififed in the Running Tests section of the [Contributor Guide].
- [ ] The title of the PR follows the [Conventional Commits] guideline
- [ ] My local branch follows the naming standards in the [Deepsource Branch Naming Convention] or [Biodiversity Branch Naming Convention]
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy],
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] All existing and new tests are passing.

[Contributor Guide]: https://github.com/FlutterPlaza/.github/blob/main/CONTRIBUTING.md
[Conventional Commits]: https://www.conventionalcommits.org/en/v1.0.0-beta.4/
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[Biodiversity Branch Naming Convention]: https://bit.ly/3DyYSwM
[Deepsource Branch Naming Convention]: https://bit.ly/3Y08Gs4

### CODE REVIEW

File: lib/core/presentation/theming/colors/colors.dart [hunk 1]
```diff
@@ -8,7 +8,7 @@ class _AppColors {
   static Color onErrorW = const Color(0xffFF000D);
   // static Color bgColorW = const Color(0xffF7F7F7);
   static Color bgColorW = Color(0xffFFFFFF);
-  static Color onBgColorW = Color(0xffD9D9D9);
+  static Color onBgColorW = Color(0xffF5F5F5);
```
Comment (fonkamloic) [inline lib/core/presentation/theming/colors/colors.dart]: Have you tested every other screen that uses this color to make sure it doesn't break? 

@DesmondTambe is responsible for setting the right color. 

Please revert this change.

Comment (DesmondTambe) [reply lib/core/presentation/theming/colors/colors.dart]: @fonkamloic right colors set already

File: lib/profile/view/profile_page.dart [hunk 2]
```diff
@@ -181,3 +72,213 @@ class _ProfileScreenState extends State<ProfileScreen> {
     );
   }
 }
+
+class AppVersion extends StatelessWidget {
```
Comment (fonkamloic) [inline lib/profile/view/profile_page.dart:76]: Move components to shared widget folder. I think it should be at core

File: lib/profile/view/profile_page.dart [hunk 3]
```diff
@@ -181,3 +72,213 @@ class _ProfileScreenState extends State<ProfileScreen> {
...
+  final BoxConstraints box;
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Column(
+        children: [
+          Text(
+            'Version 10.2', 
+            style: Theme.of(context).textTheme.titleSmall,
```
Comment (fonkamloic) [inline lib/profile/view/profile_page.dart:93]: change this to theme.

File: lib/profile/view/profile_page.dart [hunk 4]
```diff
@@ -181,3 +72,213 @@ class _ProfileScreenState extends State<ProfileScreen> {
...
+          ),
+          SizedBox(
+            height: box.maxHeight * 0.001,
+          ),
+          Text(
+            l10n.profileTermsAndConditionsText,
+            style: Theme.of(context).textTheme.titleSmall,
+          ),
+          Divider(
+            indent: (box.maxWidth / 2) - 100,
```
Comment (fonkamloic) [inline lib/profile/view/profile_page.dart:103]: what does this calculation represent?

**Review by jeffrey0606 — DISMISSED** [review]

File: lib/profile/view/profile_page.dart (hunk 2 above)
Comment (TikuGaelle) [reply lib/profile/view/profile_page.dart:76]: Okay

File: lib/profile/view/profile_page.dart (hunk 3 above)
Comment (TikuGaelle) [reply lib/profile/view/profile_page.dart:93]: Okay

File: lib/profile/view/profile_page.dart (hunk 4 above)
Comment (TikuGaelle) [reply lib/profile/view/profile_page.dart:103]: The divider does not extend to the device's entire width. 
So this calculation approximates the length of the text which is underlined.
